private/buf/cmd/buf/command/alpha/repo/reposync/reposync.go

```diff
@@ -97,11 +97,14 @@ func (f *flags) Bind(flagSet *pflag.FlagSet) {
 		),
 	)
 	// TODO: before we take this to beta, and as part of mult-module support, re-evaluate the UX of this flag
-	flagSet.StringArrayVar(
+	flagSet.StringSliceVar(
 		&f.Modules,
 		moduleFlagName,
 		nil,
-		"The module(s) to sync to the BSR; this must be in the format <module>:<module-identity>",
+		"The module(s) to sync to the BSR; this must be in the format <module-path>:<module-name>. "+
```

Comment:
Approving this PR as an incremental improvement, but we need to rethink this flag entirely per internal conversation.